src/model/wallet.ts

```diff
@@ -465,11 +615,12 @@ export const createWallet = async (
         let color = colors.getRandomColor();
         let label = '';
 
-        if (discoveredAccount) {
+        if (discoveredAccount && discoveredWalletId) {
           if (discoveredAccount.visible) {
             color = discoveredAccount.color;
-            label = discoveredAccount.label;
+            label = discoveredAccount.label ?? discoveredAccount.label;
```

Comment:
hm what's this for?

Reply:
Should be `discoveredAccount.label ?? '';`